index.html

```diff
@@ -0,0 +1,172 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+	<meta charset="UTF-8">
+	<title>Fork Me! FCC: Test Suite Template</title>
+	<link rel="stylesheet" href="css/style.css">
+	<link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet"
+		integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
+
+</head>
+
+<body>
+	<script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+	<header id="header">
+		<nav id="nav-bar" class="navbar navbar-expand-lg navbar-light bg-light fixed-top">
+			<div class="container-fluid">
+				<a class="navbar-brand" href="#" style="padding-left: 25px;">
```

Comment:
- [ ] Please avoid CSS styling inside the HMTL file. move to the css/style.css instead.

Reply:
[ ] Good job so far, :+1:  I think you can improve this message by adding a short explanation, a "why", the requested change needed to be made. This will make your comment more professional. :smile_cat:

Reply:
added `this template is more professional and complete.`